azurelinuxagent/ga/collect_logs.py

```diff
@@ -111,6 +110,7 @@ def __init__(self):
         self.should_run = True
         self.last_state = None
         self.period = conf.get_collect_logs_period()
+        self.log_collector_cgroup_path_validation_errors = 1
```

Comment:
1 makes it look like an error has happened already, I think this should init to 0